README.md

```diff
@@ -103,6 +104,6 @@ The following list shows further fiskaltrust repositories that are important for
-|[fiskaltrust/productdescription-de-doc](https://github.com/fiskaltrust/productdescription-de-doc)|Goal of this repository is to help our customers to get familiar with our products and services for the german market. It should also help our customers to apply the products and services described here.| The repository contains descriptions of the offered products and services by fiskaltrust for the german market. It also contains accompanying materials such as how-to guides, price lists, concepts and presentations, that help our customers to integrate the here offered products and services into their own products and services. |
-|[fiskaltrust/faq](https://github.com/fiskaltrust/faq)| The purpose of this repository is to provide information on frequent questions or concerns from our customers.| The repo contains unsorted, but tagged question and answer sets. If the content of the answer already exists in another public fiskaltrust repository, that content is linked in the answer. The content of this repo is also rendered and presented in the [docs.fiskaltrust](https://docs.fiskaltrust.cloud/doc/faq/qna/market-de.html) portal. |
+|[fiskaltrust/portal-manual-doc](https://github.com/fiskaltrust/portal-manual-doc)| Goal of this repository is to list and describe the available functionality of the fiskaltrust.Portal so that the usability and understanding of the portal can be increased by the here offered explanations and descriptions. | Each provided functionality of the fiskaltrust.Portal including the meaning of menu items, buttons, input fields, dropdown & checkbox values, etc., is described here. You can find different hints on how the portal helps to configure or run different fiskaltrust processes here. The description of fiskaltrust processes and products themselves are not part of this repository. |
+|[fiskaltrust/productdescription-de-doc](https://github.com/fiskaltrust/productdescription-de-doc)|Goal of this repository is to help our customers to get familiar with our products and services for the german market. It should also help our customers to apply the products and services described here.| The repository contains descriptions of the offered products and services by fiskaltrust for the german market. It also contains accompanying materials such as how-to guides, price lists, concepts, and presentations that help our customers integrate the offered products and services into their products and services. |
```

Comment:
it should also help our customers apply (not to apply)...

Since it is already being said that the goal of the repository is to help our customers get familiar (again, not TO help), i think there is no need to repeat that on the sentence that says "the repository contains descriptions of the offered..". IMO this sentence could be omitted.

Reply:
one is the goal column, one the content column. maybe not perfect from the content side, but I think it is ok for now